probe/spy.go

```diff
@@ -34,6 +35,8 @@ func spy(
 		addConnection(&r, conn, hostID, hostName)
 	}
 
+	endpoint.NatIt(hostID, r)
```

Comment:
We should guard this invocation with a check for the `conntrack` binary in the path.

```go
var nat = func(string, report.Report) {}
if endpoint.NATAvailable() {
    nat = endpoint.NAT
}

// ...

nat(hostID, r)
```